inference/sources/fly/test/fly_example.json

```diff
@@ -0,0 +1,256 @@
+{
+  "total_items": 3,
+  "page_count": 1,
+  "items": [
+    {
+      "parent_surveyid": "be5ae9dd-0189-478e-8a3d-4d8ead8240a4",
+      "parent_shortcode": "101",
+      "token": "9ba5db11ec6c63d22f08aade805bd363",
+      "surveyid": "be5ae9dd-0189-478e-8a3d-4d8ead8240a4",
+      "flowid": "100004",
+      "userid": "126",
+      "question_ref": "ref",
+      "question_idx": "10",
+      "question_text": "text",
+      "responses": [
```

Comment:
So it seems you put the format from Typeform in here. I assume that's temporary?

Reply:
Yes, it is temporary, there was a confusion, I have corrected it